docs/js-core-1/week-1/lesson.md

```diff
@@ -25,6 +23,8 @@ By the end of this class, you should be able to:
 - List and describe the different primitive data types in JS
```

Comment:
(I realise you didn't change this, but...) We don't actually teach this at all, we just define it in a glossary at the bottom... Maybe worth removing, and just not introducing the term "primitive"? We're already introducing a _lot_ of definitions i this lesson...